src/status_im/contexts/keycard/error/view.cljs

```diff
@@ -22,7 +22,9 @@
 
 (defn view
   []
-  (let [error                       (rf/sub [:keycard/application-info-error])
+  (let [error                       (or (rf/sub [:keycard/application-info-error])
+                                        :keycard/error.keycard-empty)
+        logged-in?                  (rf/sub [:multiaccount/logged-in?])
```

Comment:
empty and non empty screen should be custom in each case, so we won't have general error screen for empty and no need to have some logic here. So now we have to many issues related to this, honestly i would fix them all in one PR in this task https://github.com/status-im/status-mobile/issues/22166 which im working on

Reply:
hi @flexsurfer, Thank you very much for reviewing the PR.

> empty and non empty screen should be custom in each case

Yes, we should have a custom screen for each flow. This will make things simpler and less error-prone.

> I will fix them all in one PR for this task: https://github.com/status-im/status-mobile/issues/22166, which I am currently working on.

Thank you. Once the new custom screens are implemented, these changes will be redundant, and we can simply update the flows to use the new screens. However, for now, we need to make fixes to the already existing implementations.